content/docs/hooks-overview.md

```diff
@@ -1,107 +1,107 @@
+>Baca laman [Motivasi](/docs/hooks-intro.html#motivation) untuk mengetahui kenapa kami mengenalkan *Hooks* pada React.
 
-**↑↑↑ Each section ends with a yellow box like this.** They link to detailed explanations.
+**↑↑↑ Setiap bagian diakhiri dengan kotak kuning seperti ini.** Kotak tersebut menghubungkan ke penjelasan yang lebih terperinci.
 
-## 📌 State Hook {#state-hook}
+## 📌 Hook State {#state-hook}
 
-This example renders a counter. When you click the button, it increments the value:
+Contoh berikut me-*render* sebuah penghitung. Jika tombol di klik, nilai akan bertambah:
 
 ```js{1,4,5}
 import React, { useState } from 'react';
 
 function Example() {
-  // Declare a new state variable, which we'll call "count"
+  // Deklarasi sebuah variabel state baru, dimana akan dinamakan "count"
   const [count, setCount] = useState(0);
 
   return (
     <div>
-      <p>You clicked {count} times</p>
+      <p>Anda klik sebanyak {count} kali</p>
       <button onClick={() => setCount(count + 1)}>
-        Click me
+        Klik saya
       </button>
     </div>
   );
 }
 ```
 
-Here, `useState` is a *Hook* (we'll talk about what this means in a moment). We call it inside a function component to add some local state to it. React will preserve this state between re-renders. `useState` returns a pair: the *current* state value and a function that lets you update it. You can call this function from an event handler or somewhere else. It's similar to `this.setState` in a class, except it doesn't merge the old and new state together. (We'll show an example comparing `useState` to `this.state` in [Using the State Hook](/docs/hooks-state.html).)
+Disini, `useState` merupakan *Hook* (akan dibahas apa artinya sebentar lagi). `useState` di panggil dalam fungsi komponen untuk menambahkan suatu *state* lokal. React akan menyimpan *state* antar *render*. `useState` memberikan dua hal: nilai *state* saat ini dan fungsi untuk memperbarui nilai tersebut. Anda dapat memanggil fungsi ini dari sebuah *event handler* atau dimanapun. Hal ini serupa dengan `this.setState` pada kelas, tetapi tidak menggabungkan `state` lama dan baru menjadi satu. (Akan kami berikan contoh perbandingan `useState` dengan `this.state` pada [Menggunakan Hook State](/docs/hooks-state.html).)
 
-The only argument to `useState` is the initial state. In the example above, it is `0` because our counter starts from zero. Note that unlike `this.state`, the state here doesn't have to be an object -- although it can be if you want. The initial state argument is only used during the first render.
+Satu-satunya argumen pada `useState` yaitu nilai awal *state*. Pada contoh diatas, nilainya yaitu `0` karena counter tersebut mulai dari nol. Perlu diketahui bahwa tidak seperti `this.state`, *state* disini tidak harus berupa objek -- meskipun bisa jadi apapun yang diinginkan. Argumen nilai awal *state* hanya digunakan pada saat *render* pertama.
```

Comment:
```suggestion
Satu-satunya argumen pada `useState` yaitu nilai awal *state*. Pada contoh diatas, nilainya `0` karena penghitung mulai dari nol. Perlu diketahui bahwa tidak seperti `this.state`, *state* disini tidak harus berupa objek -- meskipun bisa jadi apapun yang diinginkan. Argumen nilai awal *state* hanya digunakan pada saat *render* pertama.
```
I think it's clearer and concise

What do you think? 😄

Reply:
Makes sense 👍